benchmarks/benchmarks/optimize.py

```diff
@@ -209,7 +209,8 @@ def bench_run(self, x0, methods=None, **minimizer_kwargs):
         if methods is None:
             methods = ["COBYLA", 'Powell',
                        'L-BFGS-B', 'BFGS', 'CG', 'TNC', 'SLSQP',
-                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-region-exact']
+                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-trlib',
```

Comment:
Of course that is not fixed. I just took this name as it is the name of the library being wrapped, so out of similarity to ``COBYLA`` and ``TNC`` I chose ``trlib``. I am not very happy with ``trust-lanczos`` although I know that it is described by this name for example in Nocedal/Wright as most of the time actually a CG algorithm is performed which is preferable out of efficiency and stability reasons to set up a Krylov subspace, then the Lanczos basis is chosen to describe the hessian in this subspace and a restricted trust region subproblem is solved in this subspace. Also knowing that CG and Lanczos algorithms are both Krylov subspace algorithms, I think there is the danger of confusing it with `trust-ncg` as the name does not exactly tell the essential differences.
So in technical terms I would prefer something like `trust-krylov-exact` but think that this is a quite long name.